File: sum.java
class sum {
    public int[] twoSum(int[] nums, int target) {
        int[] result = {0,1};
        boolean flag = false;
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                System.out.println(nums[i]+"  "+nums[j]);
                if(nums[i]+nums[j] == target){
                    flag = true;
                    result[0] = i;
                    result[1] = j;
                    break;
                }
            }
            if(flag){break;}
        }
        return result;
    }

    public static void main(String args[]){
        int[] nums = {3,2,4};
        int target = 6;
        sum t = new sum();
        int[] result = t.twoSum(nums,target);
        // System.out.println(result);
        for(int i=0;i<result.length;i++){
            System.out.println(result[i]);
        }
    }
}

File: sum.cpp
#include <vector>
#include <iostream>
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        bool flag = false;
        vector<int> result = {0,1};
        for(int i=0;i<nums.size();i++){
            for(int j=i+1;j<nums.size();j++){
                if(nums[i]+nums[j] == target){
                    flag = true;
                    result[0] = i;
                    result[1] = j;
                }
            }
            if(flag){
                break;
            }
        }
        cout<<result[0]<<endl;
        cout<<result[1]<<endl;
        return result;
    }
};

int main() 
{
    vector<int> result = {3,2,4};
    int target = 6;
    Solution s;
    s.twoSum(result,target);
    return 0;
}